### PR TITLE
changelog for Image API 2.1.1

### PR DESCRIPTION
**Pull request by tomcrane**
As close in style to the prezi one as possible

### CODE REVIEW

File: source/api/image/2.1/change-log-211.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+tags: [specifications, image-api, change-log]
+major: 2
+minor: 1
+patch: 1
+pre: final
+redirect_from:
+  - /api/image/2.1/change-log-211.html
+---
+
+This document is a companion to the [IIIF Image API Specification, Version 2.1.1][image-api]. It describes the editorial changes to the API made in this patch release. The changes are all clarifications, typo corrections or to related documents that are not [semantically versioned][semver] such as example resources and resources to manage transformation to and from the JSON-LD serialization.
```
Comment (jpstroop) [inline source/api/image/2.1/change-log-211.md]: > such as example resources and resources to manage transformation to and from the JSON-LD serialization.

...such as inline examples and resources to manage transformation to and from the JSON-LD serialization.

**Comment by azaroth42** [comment]
:+1: from me

**Comment by azaroth42** [comment]
Deployed to http://image_changelog.iiif.io/api/image/2.1/#d-change-log

**Comment by zimeon** [comment]
Fixes https://github.com/IIIF/iiif.io/issues/1153

**Pull request merged**